docs/contributing.rst

```diff
@@ -94,3 +94,27 @@ To initialize the pre-commit hook script to run in your development branch, run
 
     $ rm .git/hooks/pre-commit
 
+
+Release Branch Maintenance
```

Comment:
Do we have a policy section about what branches are considered maintained? Not sure if I can't find it, but having that alongside this would make it crystal clear what to do for contributors.

Reply:
we have the version support policy for `0.x` and `1.x`, but not for `1.8`, `1.9`, `1.10`, etc

Reply:
We don't really have a maintenance policy for the 1.x release branches. 

Our version policy can be interpreted as bug fixes should be backported to all impacted 1.x minor releases but we don't do this. We only backport up to `1.9` because ci is broken for `ddtrace<1.8`. Also backporting all the way back to say `v1.1` for a bug fix seems a bit silly.

Reply:
This is a great suggestion but should probably do it in a follow up. I don't think we should make up this policy in this PR. We should save this discussion for our midweek sync and come up with a well defined policy then.